src/accessible-autocomplete.scss

```diff
@@ -6,6 +6,7 @@ $govuk-include-default-font-face: false;
 @import "../../accessible-autocomplete/dist/accessible-autocomplete.min";
 
 .autocomplete__wrapper,
-.autocomplete__input {
+.autocomplete__input,
+.autocomplete__hint {
```

Comment:
@KatiePocklington / @kyle-bowden could you create a visual regression test for this?

Reply:
some details about how you can do it in the contributing doc https://github.com/hmrc/hmrc-frontend/blob/main/CONTRIBUTING.md#visual-regression-tests